requirements.txt

```diff
@@ -1,2 +1,2 @@
-git+https://github.com/nengo/nengo#egg=nengo
+git+https://github.com/nengo/nengo@learning-unmodulate#egg=nengo
```

Comment:
I'll see what rebasing Voja (locally only) onto this does.  We can discuss what we do next.